sqlglot/dialects/sqlite.py

```diff
@@ -75,6 +75,26 @@ def _transform_create(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
+def _generated_to_auto_increment(expression: exp.Expression) -> exp.Expression:
+    if not isinstance(expression, exp.ColumnDef):
+        return expression
+
+    generated = expression.find(exp.GeneratedAsIdentityColumnConstraint)
+
+    if generated:
+        constraints = expression.args["constraints"]
+
+        constraints.remove(generated.parent)
+
+        not_null = expression.find(exp.NotNullColumnConstraint)
+        if not_null:
+            constraints.remove(not_null.parent)
```

Comment:
My thinking was that `not_null` seems optional so it may be that a user query has `GENERATED AS IDENTITY` without `NOT NULL`, wdyt?

Reply:
It's probably ok but what I meant is you don't need to `remove` here, just do `pop` on the expressions you get back from `find` since you're checking that they're not falsey.

Reply:
Ah right, misinterpreted the comment